playbooks/roles/openvpn/templates/client-common.j2

```diff
@@ -7,6 +7,7 @@ nobind
 persist-key
 persist-tun
 ns-cert-type server
+tls-version-min 1.2
```

Comment:
Tested with OpenVPN connect on iOS, as well as viscosity on windows and mac os x.

Viscosity was compiled with OpenVPN 2.3.10 at the time, no issues whatsover. iOS worked just fine.